src/components/identicon/Identicon.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+import ReactIdenticon from 'react-identicons'
+import { colors } from 'ipfs-css/theme.json'
+
+const identiconPalette = [colors.navy, colors.aqua, colors.gray, colors.charcoal, colors.red, colors.yellow, colors.teal, colors.green]
+
+const Identicon = ({ size = 20, cid }) => <ReactIdenticon string={cid} size={size} palette={identiconPalette} />
```

Comment:
We should downscale the default size as with `20` the icon is too large in my opinion.

`12` works well for me, although I'd say it will feel a bit small for some.

Reply:
I changed the default to `14`, it seems like a good middle-ground and it matches the font size of the CIDs in the Peer Table.